### PR TITLE
Readme updates.

### PR DESCRIPTION
**Pull request by JonathanHenson**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,67 @@ AWS Crypto Abstraction Layer: Cross-Platform, C99 wrapper for cryptography primi
...
+````
+
+## FAQ
+### I want more algorithms, what do I do?
+Great! So do we! At a minimum, file an issue letting us know. If you want to file a Pull Request, we'd be happy to review and merge it when it's ready.
+### When will this be in the language specific crt packages?
+As soon as we can possibly get it done.
+### Who should consume this package directly?
+Are you writing C directly? Then you should.
+Are you using any other programming language? This functionality will be exposed via. that language specific crt packages.
```
Comment (justinboswell) [inline README.md]: via doesn't need a period

File: README.md [hunk 2]
```diff
@@ -4,4 +4,67 @@ AWS Crypto Abstraction Layer: Cross-Platform, C99 wrapper for cryptography primi
...
+
+##### One-Shot
+````
+aws_sha256_hmac_compute(allocator, &secret_buf, &your_buffer, &output_buffer, 0);
+````
+
+## FAQ
+### I want more algorithms, what do I do?
+Great! So do we! At a minimum, file an issue letting us know. If you want to file a Pull Request, we'd be happy to review and merge it when it's ready.
+### When will this be in the language specific crt packages?
```
Comment (justinboswell) [inline README.md]: I would just leave this one out, even though it doesn't give dates.

File: README.md [hunk 3]
```diff
@@ -4,4 +4,67 @@ AWS Crypto Abstraction Layer: Cross-Platform, C99 wrapper for cryptography primi
 
 ## License
 
-This library is licensed under the Apache 2.0 License. 
+This library is licensed under the Apache 2.0 License.
+
+## Supported Platforms
```
Comment (justinboswell) [inline README.md]: What about build instructions?

File: README.md (hunk 2 above)
Comment (JonathanHenson) [reply README.md]: okay

File: README.md (hunk 3 above)
Comment (JonathanHenson) [reply README.md]: sure

**Pull request merged**